lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -1020,9 +1113,32 @@ def ignore_case_applies?(ignore_case, key)
       false
     end
 
+    # takes an operation that can be any of:
+    #   - plain string identifier
+    #   - a PolicyMachineStorageAdapter::ActiveRecord::Operation instance
+    #   - a PM::Operation instance
+    # and returns its identifier (aka name)
+    def operation_identifier(operation)
+      operation.try(:unique_identifier) || operation.to_s
+    end
+
+    # gives the theoretical prohibition identifier (name) for a given operation
+    # NOTE: does not confirm that this prohibition exists like the funcs below
+    def prohibition_id(operation)
```

Comment:
For consistency, how about calling this `prohibition_identifier` ?

Reply:
sure 👍